src/unittest/test_utilities.cpp

```diff
@@ -636,3 +638,69 @@ void TestUtilities::testSanitizeDirName()
 	UASSERT(sanitizeDirName("cOnIn$", "~") == "~cOnIn$");
 	UASSERT(sanitizeDirName(" cOnIn$ ", "~") == "_cOnIn$_");
 }
+
+template <typename F, typename C>
+C container_map(const C &co, F functor)
+{
+	C ret;
+	for (auto it = co.begin(); it != co.end(); it++)
+		ret.push_back(functor(*it));
+	return ret;
+}
+
+void TestUtilities::testIsBlockInSight()
+{
+	std::vector<v3s16> testdata1 = {
```

Comment:
Should be made `const` to catch modification attempts at compile time.
Also how about using `v3f32`? That would make the manual conversion in L667 obsolete.

Reply:
1. done.
2. I would then have to cast all the other cases, since the blockpos needs to be v3s16